### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by alexet**
Add issue templates for creating a release checklist and also standardish templates for bug and feature reports.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-extension-release.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+---
+
+- [ ] Trigger a release build on Actions by adding a new tag on master of the format `vxx.xx.xx`
+- [ ] Monitor the status of the release build in the `Release` workflow in the Actions tab.
+- [ ] Download the VSIX from the draft GitHub release that is created when the release build finishes.
+- [ ] Log into the [Visual Studio Marketplace](https://marketplace.visualstudio.com/manage/publishers/github).
+- [ ] Click the `...` menu in the CodeQL row and click **Update**.
+- [ ] Drag the `.vsix` file you downloaded from the GitHub release into the Marketplace and click **Upload**.
+- [ ] Publish the GitHub release.
```
Comment (adityasharad) [inline .github/ISSUE_TEMPLATE/new-extension-release.md]:
```suggestion
- [ ] Publish the draft GitHub release and confirm the new release is marked as the latest release at https://github.com/github/vscode-codeql/releases.
```
Adding @henrymercer's recent edits.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (henrymercer) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Suggest adding the "bug" label here

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (henrymercer) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Suggest adding the "enhancement" label here

File: .github/ISSUE_TEMPLATE/new-extension-release.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+name: New extension release
+about: Create an issue with a checklist for the release steps (write access required
+  for the steps)
+title: Release Checklist for version xx.xx.xx
+labels: ''
+assignees: ''
+
+---
+
```
Comment (henrymercer) [inline .github/ISSUE_TEMPLATE/new-extension-release.md]: Suggest adding "Update the version number in the issue title" as the first step?

**Pull request merged**